awx/ui_next/src/components/CodeMirrorInput/VariablesField.jsx

```diff
@@ -20,7 +20,15 @@ const StyledCheckboxField = styled(CheckboxField)`
   --pf-c-check__label--FontSize: var(--pf-c-form__label--FontSize);
 `;
 
-function VariablesField({ i18n, id, name, label, readOnly, promptId }) {
+function VariablesField({
+  i18n,
```

Comment:
On a new WF Template Form:
1. Go to variables
2. Set some vars in the YAML field, like `foo: bar`. Save.
3. Go to edit the field again.

The current value vanishes, as opposed to having kept it for editing.

Reply:
This sounds like a legitimate bug.  If this is unique to WFJT's then lets fix it in this branch.  If it impacts other extra vars in other forms then lets file a bug and move forward.

Reply:
yeah that was a legitimate bug and has been resolved.

Reply:
@AlexSCorey this bug was within the latest branch I pulled this morning. Is it resolved in another branch? If so, could you rebase so that the bug is fixed here? thanks!

Reply:
This has been fixed, thanks.